src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5244,11 +5244,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ConditionalInInterpolation" xml:space="preserve">
     <value>A conditional expression cannot be used directly in a string interpolation because the ':' ends the interpolation. Parenthesize the conditional expression.</value>
   </data>
-</root>
```

Comment:
How did we miss this misplaced tag? Is there a test gap?

Reply:
The branch previously wasn't running tests and Vlad asked me to push whatever I had (even if it didn't build) so he could look at the progress. This commit should bring back full testing and bring the feature branch into a good state.